Assets/MixedRealityToolkit.Services/InputSystem/InputSystemGlobalHandlerListener.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// </summary>
     public abstract class InputSystemGlobalHandlerListener : MonoBehaviour
     {
-        private bool lateInitialize = true;
+        private bool lateInitialize = false;
```

Comment:
@googlan Is this a bugfix? Doesn't look related to the rest of the changes here.

Reply:
Yeah it's a bugfix, for #5029. I need it for the tests to pass. I can make it as a separate PR, but didn't want to add extra overhead of getting approve for such a small change.

Reply:
I found the commit that introduced the lateInitialize variable:

```
1c17f7847e1d3a3a8e125b876a9c57ad1fb42a6f
Fixed input handler race condition if input system and teleport system not found before gameobjects wake up
```

I think with my change, the handler will never register itself if the input system is not available when the component gets enabled. I think we need one more change. I'll go ahead and pull that as separate pr.

Reply:
Moved to separate PR.